mir-ci/mir_ci/test_drag_and_drop.py

```diff
@@ -1,14 +1,18 @@
-import asyncio
+import tempfile
 from pathlib import Path
 
 import pytest
 from mir_ci import SLOWDOWN, apps
 from mir_ci.display_server import DisplayServer
-from mir_ci.virtual_pointer import Button, VirtualPointer
+from mir_ci.virtual_pointer import VirtualPointer
 
-APP_PATH = Path(__file__).parent / "clients" / "drag_and_drop_demo.py"
+MIR_CI_PATH = str(Path(__file__).parent)
+APP_PATH = MIR_CI_PATH + "/clients/drag_and_drop_demo.py"
 STARTUP_TIME = 1.5 * SLOWDOWN
 A_SHORT_TIME = 0.3
+ROBOT_HEADER = f"""*** Settings ***
+Library    {MIR_CI_PATH}/robot/libraries/WaylandHid.py
+"""
```

Comment:
Let's build a template here:

```suggestion
ROBOT_TEMPLATE = f"""*** Settings ***
Library    {MIR_CI_PATH}/robot/libraries/WaylandHid.py

*** Test Cases ***
{{test_case}}
"""
```

I imagine longer term we'll build it out to a class of its own, wrapping the temporary file handling etc., but this will do for now.